fastai/utils/ipython.py

```diff
@@ -16,6 +16,13 @@ def is_in_ipython():
 
 IS_IN_IPYTHON = is_in_ipython()
 
+def is_in_colab():
+    "Is the code running in Google Colaboratory?"
+    if IS_IN_IPYTHON and os.path.exists('/var/colab'): return True
```

Comment:
Hoping nobody happens to have this path for whatever other reason - perhaps not the best check.
Here https://stackoverflow.com/a/39662359/9201239 in comments someone proposed:

> If someone is interested in detecting whether the notebook is running on Google Colab you can check this: get_ipython().__class__.__module__ == "google.colab._shell" – guiferviz Nov 19 '18 at 22:04

Reply:
Yes much better, thank you!